src/interfaces/node.cpp

```diff
@@ -87,6 +87,12 @@ class NodeImpl : public Node
         }
     }
     bool getProxy(Network net, proxyType& proxy_info) override { return GetProxy(net, proxy_info); }
+
+    bool hasTorOnlyConnections() override
```

Comment:
would it make sense to expose this bool over rpc as well?

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/86#issuecomment-690228628).

Reply:
See https://github.com/bitcoin/bitcoin/pull/20172.